7_react/react1/week1/ToDoList/src/Components/TaskInput.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from "react";
+import "../index.css";
+
+function TaskInput() {
+  const [toDoList, setToDoList] = useState([]);
+  const [todo, setTodo] = useState("");
+
+  console.log(todo);
+
+  function handleSubmit(e) {
+    e.preventDefault();
+
+    const newTodo = {
+      id: new Date().getTime(),
+      text: todo,
+      completed: false,
+    };
+    setToDoList([...toDoList].concat(newTodo));
```

Comment:
Good job, although this is also ok, it's still safer to use previous data from `set` method itself like 
`setToDoList(prevTodos => [ ...prevTodos, newTodo ] );`